samples/wifi/raw_tx_packet/README.rst

```diff
@@ -0,0 +1,173 @@
+.. _wifi_raw_tx_packet_sample:
+
+Wi-Fi: Raw TX packet
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Raw TX packet sample demonstrates how to transmit raw IEEE 802.11 packets.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+The sample generates and broadcasts 802.11 beacon frames as raw TX packets.
+As a consequence, the nRF70 Series device can be identified as a Wi-Fi® beaconing device.
+
+The sample demonstrates how to transmit raw TX packets in both connected Station and non-connected Station modes of operation.
+The sample provides the option to select the traffic pattern between the following modes:
+
+* Continuous packet transmission.
+* Fixed number of transmitted packets.
+
+The configurations for connected Station or non-connected Station modes, and for continuous or fixed packet transmission, are set at build time.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/raw_tx_packet/Kconfig`):
+
+.. options-from-kconfig::
+
+By using the following configuration options, you can configure the sample for different operational modes, including specific raw TX packet header settings:
+
+Configuration options for operating modes
+-----------------------------------------
+
+  * For connected Station mode
+
+    To configure the sample in connected Station mode, you must configure the following Wi-Fi credentials in the :file:`prj.conf` file:
+
+    * ``Service Set Identifier (SSID)``: Defines the name of your Wi-Fi network.
```

Comment:
please use `:ref:kconfig:option ...`

Reply:
We could also use :ref: to link to the Kconfig options which are mentioned in the previous section "Configuration options".
For example: see this: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12872/nrf/samples/wifi/sr_coex/README.html#configuration-files